packages/nimble-components/src/table/components/cell/styles.ts

```diff
@@ -6,7 +6,15 @@ export const styles = css`
     ${display('grid')}
 
     :host {
+        --ni-private-table-cell-offset-factor: 0;
         padding: 0px calc(${standardPadding} / 2);
+        padding-left: calc(
+            ${standardPadding} / 2 +
+                (
+                    var(--ni-private-table-cell-offset-factor) *
```

Comment:
Maybe `--ni-private-table-cell-offset-count`? I think count conveys the incremental integer nature a bit better than factor.

Edit: Actually Molly's nesting level is even better, `--ni-private-table-cell-nesting-level`